core/test-runtime/client/src/lib.rs

```diff
@@ -157,6 +177,13 @@ pub trait TestClientBuilderExt<B>: Sized {
 	/// Override the default value for Wasm heap pages.
 	fn set_heap_pages(self, heap_pages: u64) -> Self;
 
+	/// Add an extra value into the genesis storage.
+	///
+	/// # Panics
+	///
+	/// Panics if the key is empty.
+	fn add_extra_storage(self, key: Vec<u8>, value: Vec<u8>) -> Self;
```

Comment:
```suggestion
	fn add_extra_storage<K: Into<Vec<u8>, V: Into<Vec<u8>>(self, key: K, value: V) -> Self;
```

So the api is more ergonomic.

Reply:
Agreed!